datahub-web-react/src/app/ingest/source/ExecutionRequestDetailsModal.tsx

```diff
@@ -1,48 +1,147 @@
+    const resultText = result && (
+        <Typography.Text style={{ color: resultColor, fontSize: 14 }}>
+            {ResultIcon && <ResultIcon style={{ marginRight: 4 }} />}
+            {getExecutionRequestStatusDisplayText(result)}
+        </Typography.Text>
+    );
+    const resultSummaryText =
+        (result && (
+            <Typography.Text type="secondary">
+                Ingestion {result === 'SUCCESS' ? 'successfully completed' : 'completed with errors'}.
+            </Typography.Text>
+        )) ||
+        undefined;
+
     return (
         <Modal
-            width={1300}
-            footer={null}
-            title={<Typography.Text>Execution Details</Typography.Text>}
+            width={800}
+            footer={
+                <>
+                    <Button onClick={onClose}>Close</Button>
+                </>
+            }
+            style={{ top: 100 }}
+            bodyStyle={{ padding: 0 }}
+            title={
+                <HeaderSection>
+                    <Typography.Title style={{ padding: 0, margin: 0 }} level={4}>
+                        Ingestion Run Details
+                    </Typography.Title>
+                </HeaderSection>
+            }
             visible={visible}
             onCancel={onClose}
         >
             {!data && loading && <Message type="loading" content="Loading execution details..." />}
             {error && message.error('Failed to load execution details :(')}
             <Section>
-                <SectionHeader strong>Captured Output</SectionHeader>
-                <Typography.Paragraph ellipsis>
-                    <pre>{`${output}`}</pre>
-                </Typography.Paragraph>
+                <StatusSection>
+                    <Typography.Title level={5}>Status</Typography.Title>
+                    <div style={{ marginBottom: 4 }}>{resultText}</div>
+                    <div>{resultSummaryText}</div>
+                </StatusSection>
+                {result === 'SUCCESS' && (
+                    <IngestedAssetsSection>
+                        {data?.executionRequest?.id && <IngestedAssets urn={urn} id={data?.executionRequest?.id} />}
+                    </IngestedAssetsSection>
+                )}
+                <LogsSection>
+                    <SectionHeader level={5}>Logs</SectionHeader>
+                    <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+                        <Typography.Paragraph style={{ marginBottom: 0 }} type="secondary">
+                            View logs that were collected during the ingestion run.
+                        </Typography.Paragraph>
+                        <Button type="text" onClick={downloadLogs}>
+                            <DownloadOutlined />
+                            Download
+                        </Button>
+                    </div>
+                    <Typography.Paragraph ellipsis>
+                        <pre>{`${logs}...`}</pre>
```

Comment:
you only want to add the ellipses here if `showExpandedLogs` is false right?

Reply:
Yes!